### PR TITLE
feature/#257_Try_vertical_tabs_similar_to_the_Capabilities_tab

### PR DESCRIPTION
**Pull request by olatechpro**
Feature/#257 try vertical tabs similar to the capabilities tab

editor-tabs

### CODE REVIEW

**Comment by olatechpro** [comment]
@agapetry due to changes to Editor features display post type, there'll be need to adjust PRO custom item display to match the number of row but i can only make this changes after this PR is approved and merge to PRO unless you want me to make a copy of this PR for pro and you can review both seperately.

**Review by agapetry — APPROVED** [review]

**Pull request merged**
